src/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Windows.cs

```diff
@@ -58,7 +58,7 @@ internal override object PostCompletion(int numBytes)
                         SocketOptionLevel.Socket,
                         SocketOptionName.UpdateAcceptContext,
                         ref handle,
-                        Marshal.SizeOf(handle));
+                        IntPtr.Size);
```

Comment:
There are multiple issues on this already:

https://github.com/dotnet/roslyn/issues/15116
https://github.com/dotnet/roslyn/issues/3208

I do not think we need yet another one.